Makefile

```diff
@@ -60,3 +62,27 @@ source/css/sabre.css: source/less/*.less
 
 foo:
 	echo $(SOURCE_MD_FILES)
+
+clean:
+	rm -Rvf output_dev/ source/components/* vendor/ source/*.css
+
+
+### If no PHP configuration changes to the current dev host are desired
+### or allowed and container support is enabled:
+
+DOCKER_ENABLED=$(shell which docker; echo $$?)
```

Comment:
I'd definitely like to keep docker out of this repo. Maybe for you this means to create a separate repository that builds the site for you with docker. Sorry about that =(